pkg/daemon/update.go

```diff
@@ -1164,6 +1164,11 @@ func (dn *Daemon) updateFiles(oldIgnConfig, newIgnConfig ign3types.Config) error
 	if err := dn.writeUnits(newIgnConfig.Systemd.Units); err != nil {
 		return err
 	}
+	// In OKD /var/home/core/.ssh/authorized_keys is not created during ignition.  Create
+	// from /var/home/core/.ssh/authorized_keys.d/ignition if missing
+	if err := dn.writeMissingAuthorizedKeys(); err != nil {
+		return err
+	}
```

Comment:
@vrutkovs this commit can be dropped in the next rebase now that https://github.com/openshift/machine-config-operator/pull/2688 has merged